tests/lvm.spec.ts

```diff
@@ -11,33 +15,30 @@ test('Use logical volume management (LVM) as storage device for installation', a
   });
   await page.getByRole('link', { name: 'Storage' }).click();
 
-  //Storage page
+  // POMed code
```

Comment:
only `non-POM` code comment is neede, the POM code doesn't need a comment because it is default, so we don't need to remove that later.

Reply:
Yes but it's not clear just by blank line if the Non-POMed code is finished. Anyway, I would suggest we don't use this type of comment at all, as the progress tickets descriptions should be clear enough for us to understand what we need to replace